docs/README.md

```diff
@@ -34,7 +34,10 @@
 - [Federation](./guides/federation.md)
 - [Custom extensions](./guides/custom-extensions.md)
 - [File upload](./guides/file-upload.md)
-- [Pagination](./guides/pagination.md)
+- [Pagination](./guides/pagination/overview.md)
+  - [Offset Pagination](./guides/pagination/offset-based.md)
+  - [Cursor Pagination](./guides/pagination/cursor-based.md)
+  - [Working with Connections](./guides/pagination/connections.md)
```

Comment:
Can we refer the Relay here to improve SEO (search engine optimization)? Relay Pagination for example

Reply:
Sure! We could do that!

Reply:
done!